public/components/custom_panels/helpers/custom_modals/custom_input_modal.tsx

```diff
@@ -0,0 +1,99 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ *
+ * Modifications Copyright OpenSearch Contributors. See
+ * GitHub history for details.
+ */
+
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
FYI i don't think this license header is required unless the same file existed in ODFE repo

Reply:
Will remove this one. The file has many things common with ODFE repo "notebooks" modal container. But will add more modal components going forward.